src/index.html

```diff
@@ -2,13 +2,45 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.google.com/specimen/Roboto" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1>Moyo Header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo"
+         class="header_img">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item"><a class="nav_link is-active" href="Apple">Apple</a></li>
+          <li class="nav_item"><a class="nav_link" href="Samsung">Samsung</a></li>
+          <li class="nav_item"><a class="nav_link" href="Smartphones">Smartphones</a></li>
```

Comment:
```suggestion
          <li class="nav_item">
            <a class="nav_link is-active" href="Apple">Apple</a>
          </li>
          
          <li class="nav_item">
            <a class="nav_link" href="Samsung">Samsung</a>
          </li>
          
          <li class="nav_item">
            <a class="nav_link" href="Smartphones">Smartphones</a>
          </li>
          
```
for increase readibility better to make empty lines between siblings  (items in list), but not between child and parent